### PR TITLE
Backport 1.8.2

### PR DESCRIPTION
**Pull request by tnix100**

### CODE REVIEW

**Review by Bloctans — APPROVED** [review]

**Review by HAZARD6896 — CHANGES_REQUESTED** [review]
The z-index of the drop-down menu needs to be set to the front because it often doesn’t appear on mobile. Last request before I approve the commit

**Review by showierdata9978 — APPROVED** [review]
lgtm

**Comment by roblox888i235** [comment]
New settings page should be in this update ngl

**Comment by showierdata9978** [comment]
> New settings page should be in this update ngl

No, that does not have mobile support.

**Comment by roblox888i235** [comment]
we could do that

**Review by showierdata9978 — DISMISSED** [review]
profile picture is squished on mobile 
when debug menu is enabled

File: src/lib/Post.svelte
Comment (CST1229) [inline src/lib/Post.svelte:580]: this probably won't work in light mode or other themes (maybe just change the opacity instead)

File: src/lib/modals/moderation/ModPanel.svelte
Comment (roblox888i235) [inline src/lib/modals/moderation/ModPanel.svelte:257]: should be in 256 line

**Review by roblox888i235 — APPROVED** [review]

File: src/lib/modals/moderation/ModPanel.svelte
Comment (CST1229) [reply src/lib/modals/moderation/ModPanel.svelte:257]: if you mean the positioning of that one `>`, this is done by the formatter we're using (i assume it's trying to make some of the `</button>` be unindented without adding any white space to the element which might affect things)

Comment (showierdata9978) [reply src/lib/modals/moderation/ModPanel.svelte:257]: yeah thats from prettier

**Review by zedsleepyhead — COMMENTED** [review]
This isn't specific to the changes made, but I just noticed Meower isn't capitalized here for some reason

image

**Comment by tnix100** [comment]
> This isn't specific to the changes made, but I just noticed Meower isn't capitalized here for some reason
> 
> image

It's an Ocular reference, so all lower-case makes sense.

**Comment by meltland2002** [comment]
> > This isn't specific to the changes made, but I just noticed Meower isn't capitalized here for some reason
> > image
> 
> It's an Ocular reference, so all lower-case makes sense.

but it's improper

**Comment by zedsleepyhead** [comment]
> > This isn't specific to the changes made, but I just noticed Meower isn't capitalized here for some reason
> > image
> 
> It's an Ocular reference, so all lower-case makes sense.

oops I didn't pick up on that

**Comment by tnix100** [comment]
> > > This isn't specific to the changes made, but I just noticed Meower isn't capitalized here for some reason
> > > image
> > 
> > 
> > It's an Ocular reference, so all lower-case makes sense.
> 
> but it's improper

Yeah, it is, but that's intentional and the change to make it proper has been declined by CST in a previous pull request.

**Review by zedsleepyhead — APPROVED** [review]

**Comment by meltland2002** [comment]
> > > > This isn't specific to the changes made, but I just noticed Meower isn't capitalized here for some reason
> > > > image
> > > 
> > > 
> > > It's an Ocular reference, so all lower-case makes sense.
> > 
> > 
> > but it's improper
> 
> Yeah, it is, but that's intentional and the change to make it proper has been declined by CST in a previous pull request.

womp womp

**Comment by tnix100** [comment]
> profile picture is squished on mobile when debug menu is enabled

It isn't that squished for me unless I also have the admin panel. It's just an issue of having too many menu items, but a general user won't have debug or the admin panel.

File: src/lib/Post.svelte
Comment (tnix100) [reply src/lib/Post.svelte:580]: Oops, that is pretty bad. I will fix.

**Comment by showierdata9978** [comment]
> > profile picture is squished on mobile when debug menu is enabled
> 
> It isn't that squished for me unless I also have the admin panel. It's just an issue of having too many menu items, but a general user won't have debug or the admin panel.

that's an issue when there is no way to disable menu item(s) from the main client

**Comment by roblox888i235** [comment]
Just a question, where do i import https://github.com/meower-community/Meowerbot-Scratch/blob/main/DefaultBot.sb3

**Comment by showierdata9978** [comment]
> > > profile picture is squished on mobile when debug menu is enabled
> > 
> > 
> > It isn't that squished for me unless I also have the admin panel. It's just an issue of having too many menu items, but a general user won't have debug or the admin panel.
> 
> that's an issue when there is no way to disable menu item(s) from the main client

.

**Review by HAZARD6896 — APPROVED** [review]
Gg

**Review by showierdata9978 — APPROVED** [review]

**Comment by tnix100** [comment]
I'm going to merge now without approval from Mike or CST. CST only requested one change which has been done.

**Pull request merged**
